bamboo/generate-ts-build-cache.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -ex
 
+. ./bamboo/set-bamboo-env-variables.sh
+. ./bamboo/abort-if-not-pr.sh
+
 NONCACHE_WORKING_DIR=$(pwd)
 CURRENT_WORKING_DIR=$NONCACHE_WORKING_DIR
 
```

Comment:
Hmm, so this wasn't using the cached bootstrap.   I wonder if that speeds up the first step?